src/main/java/TaskList.java

```diff
@@ -0,0 +1,30 @@
+import java.util.*;
+
+public class TaskList {
+    public static ArrayList <Task> arr = new ArrayList <Task>();
```

Comment:
I think it would be better if we rename this variable to 'tasks' or a similar name to improve readability, as this variable is a class variable and will be used often.